core/src/main/java/org/fao/geonet/kernel/datamanager/base/BaseMetadataUtils.java

```diff
@@ -295,7 +295,8 @@ public String extractDefaultLanguage(String schema, Element md) throws Exception
     public LinkedHashMap<String, String> extractTitles(@Nonnull String id) throws Exception {
         AbstractMetadata metadata = findOne(id);
 
-        if (metadata == null)
+        // If metadata not found or it is not a type metadata then return null
+        if (metadata == null || metadata.getDataInfo().getType() != MetadataType.METADATA)
```

Comment:
Should we also keep this for templates ?

Reply:
@fxprunayre
I have applied a fix so that it also include templates.